package.json

```diff
@@ -107,7 +107,8 @@
     "cross-fetch": "^2.2.6",
     "glob-parent": "^6.0.2",
     "parse-url": "^8.1.0",
-    "graphql": "15.8.0"
+    "graphql": "15.8.0",
+    "xml2js": "0.5.0"
```

Comment:
Why is the new dep necessary?

Reply:
It is one of the downstream dependency of `aws-sdk`. The alternative is to bump `aws-sdk` instead

Reply:
Oh I didn't realize this was in `resolutions`. Got it.